src/widgetresize/resizerstate.js

```diff
@@ -126,11 +126,8 @@ export default class ResizeState {
 
 		const widthStyle = domResizeHost.style.width;
 
-		if ( widthStyle && widthStyle.match( /^\d+\.?\d*%$/ ) ) {
-			this.originalWidthPercents = parseFloat( widthStyle );
-		} else {
-			this.originalWidthPercents = this._getParentPercentage( domResizeHost );
-		}
+		this.originalWidthPercents = widthStyle && widthStyle.match( /^\d+\.?\d*%$/ ) ?
```

Comment:
Why this change? It makes the code harder to read.

Reply:
Is it really that harder to read? See the code in full context. Note that it now replaces:

```js
if ( widthStyle && widthStyle.match( /^\d+\.?\d*%$/ ) ) {
	this.originalWidthPercents = parseFloat( widthStyle );
} else {
	this.originalWidthPercents = calculateHostPercentageWidth( domResizeHost );
}
```

Rather than the previous form.

Reply:
Yes, it is. Because if-else is *the* way to write a condition. If you see it, you know there's a condition. Your current code must be first parsed by the reader to understand what's happening.